task3/index.html

```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Gektor</title>
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+        <body>
+            <div class="leftcol">
+                <header>
+                    <img class="logo" src="logo.jpg" align="center" alt="logotip"> <strong>Gektor</strong>
+                    <p class="username"><strong>Anastasiya Zyatkova</strong></p>
+                </header>
+                <div id="menu">
+                    <p><img class="picbut" src="home.png" alt="home" align="center"><button class="menubut"><strong>Home</strong></button></p>
+                    <p><img class="picbut" src="profile.png" alt="profile" align="center"><button class="menubut"><strong>Profile</strong></button></p>
```

Comment:
The strong tag means the text is important.
So possibly menu is important thing but we shouldn't mark that in markup :)
Use css for stylistic, html should reflect the meaning (semantic).